lib/submariner_prepare/validate_acm_readiness.sh

```diff
@@ -181,7 +206,9 @@ function check_clusters_deployment() {
     detect_sno_clusters
 
     if [[ "$PLATFORM" =~ "vsphere" ]]; then
-        check_available_vsphere_platform_clusters
+        check_managed_clusters_available_platform "vsphere"
+    elif [[ "$PLATFORM" =~ "aro" ]]; then
```

Comment:
When using regex compression, you don't need to quote the value after =~ "regex value"